### PR TITLE
Test turn generation via goldenfiles instead. Revert the "import from a

### PR DESCRIPTION
**Pull request by dabreegster**
raw string" stuff from the previous commit.

@michaelkirk How's this?

### CODE REVIEW

File: abstutil/src/lib.rs [hunk 1]
```diff
@@ -68,6 +68,8 @@ lazy_static::lazy_static! {
             "data".to_string()
         } else if file_exists("../data/".to_string()) {
             "../data".to_string()
+        } else if file_exists("../../data/".to_string()) {
```
Comment (dabreegster) [inline abstutil/src/lib.rs:71]: So you can run `cargo run` from within `map_tests/src`

**Comment by dabreegster** [comment]
Merging to avoid diffbasing, but feedback still welcome

File: data/regen.sh [hunk 2]
```diff
@@ -14,3 +14,7 @@ rm -fv data/system/maps/huge_seattle.bin data/input/raw_maps/huge_seattle.bin da
 cargo run --release --bin game -- --prebake
 cargo run --release --bin game -- --smoketest
 cargo run --release --bin game -- --check_proposals
+
+# This is cheap enough to do before every single commit, but since we don't
+# have presubmit tests, it's fine to just run it here.
```
Comment (michaelkirk) [inline data/regen.sh:19]: In some projects I check in some suggested git hooks. e.g.:

```
scripts/git_hooks/precommit
scripts/git_hooks/postcommit
```

Then locally developers can do something like
```
for file in ls scripts/git_hooks/*; do
    ln -s $file .git/hooks
done
```

Honestly, it's kind of janky, but it's the best I can suggest. 😅

File: map_tests/goldenfiles/left_turn_and_bike_lane.txt [hunk 3]
```diff
@@ -0,0 +1,49 @@
+TurnID(Lane #0, Lane #5, Intersection #0) is a Crosswalk
```
Comment (michaelkirk) [inline map_tests/goldenfiles/left_turn_and_bike_lane.txt:1]: so readable!

File: data/regen.sh (hunk 2 above)
Comment (dabreegster) [reply data/regen.sh:19]: That's a nifty trick. :P  I'll try this out and see how it works. I use `git commit --amend` constantly in my local workflow and don't want to trigger it then. There's also a very low risk of accidentally breaking turn generation without realizing it; it's pretty obvious when a PR is touching relevant code and requires map data regeneration.

**Review by michaelkirk — COMMENTED** [review]
This whole approach seems really promising! Nice work.

File: data/regen.sh (hunk 2 above)
Comment (michaelkirk) [reply data/regen.sh:19]: I guess something like this should usually work:

https://stackoverflow.com/a/56955369/353178

```
IS_AMEND=$(ps -ocommand= -p $PPID | grep -e '--amend');

if [ -n "$IS_AMEND" ]; then
  return;
fi

do_the_thing
```

Comment (michaelkirk) [reply data/regen.sh:19]: I'm also a heavy ammender: `alias gca='git commit --amend'` =)

**Pull request merged**